packages/cna-template/template/tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
```

Comment:
any reason why it's not true by default?

Reply:
This is the `tsconfig.json` that's generated by Next.js. Not sure why it defaults to `false` here, but I think you're right that we should change it to `true`.

Reply:
Done in 46c3699.